lib/helpers.js

```diff
@@ -12,7 +13,7 @@ export function notifyAllowWallet({account, preventRenotify = false}) {
     return;
   }
   // if inside a WebView ignore
-  if(isWebView({userAgent: navigator?.userAgent})) {
+  if(config?.isWebView === true) {
```

Comment:
I'd suggest renaming this to reflect the behavior it has.  "isWebView" suggests this is here to handle a specific current platform issue.  How about something like "disableCHAPI"?  (Not sure if that is correct, but that's the idea. 
 Maybe "disableCHAPINotifications"?)  Then the code that checks the platform can set flags as needed depending on what is supported.

Also can simplify to check truthiness like `if(config?.xxxx)`.

Reply:
Addressed here: https://github.com/digitalbazaar/bedrock-vue-wallet/pull/36/commits/df92b0ed8a870a798b35f16997a0c9858d2abdf0

I did use `disableChapi` instead of `disableCHAPI` if you want to correct feel free.